### PR TITLE
Fetch only ongoing dispatches

### PR DESCRIPTION
**Pull request by Marenz**
Based on https://github.com/frequenz-floss/frequenz-client-dispatch-python/pull/212

### CODE REVIEW

File: tests/test_frequenz_dispatch.py [hunk 1]
```diff
@@ -524,23 +523,31 @@ async def test_notification_on_actor_start(
...
 
-    fake_time.shift(timedelta(seconds=1))
-    await asyncio.sleep(1)
+    fake_time.shift(timedelta(seconds=7))
+    await asyncio.sleep(6)
+    logging.warning("Time shifted")
 
     # Expect notification of the running dispatch being ready to run
     ready_dispatch = await test_env.running_state_change.receive()
+    logging.warning("Received running state change: %s", ready_dispatch)
```
[the remark this answers is not in the document]
Comment (Marenz) [reply tests/test_frequenz_dispatch.py]: Removed logging

File: tests/test_frequenz_dispatch.py [hunk 2]
```diff
@@ -476,9 +476,7 @@ async def test_dispatch_new_but_finished(
     )
     fake_time.shift(timedelta(seconds=1))
 
-    await asyncio.sleep(1)
-    # Process the lifecycle event caused by the old dispatch at startup
-    await test_env.lifecycle_events.receive()
+    # Expecting no event for a past and finished dispatch as it should be filtered out
```
Comment (llucax) [inline tests/test_frequenz_dispatch.py:479]: What about the 5 seconds in the past? Wouldn't it be possible to get already finished dispatches in some edge cases?

**Review by llucax — COMMENTED** [review]
Nice, LGTM, no approving only because of the PR dependency.

File: tests/test_frequenz_dispatch.py (hunk 2 above)
Comment (Marenz) [reply tests/test_frequenz_dispatch.py:479]: Yes, but that's not an issue, we still manage/filter them on our side, the server-filter is just an optimization

Comment (Marenz) [reply tests/test_frequenz_dispatch.py:479]: And we rather have too much than too little, thus the 5 second buffer

**Review by llucax — APPROVED** [review]
